### PR TITLE
Combine moreores into coreores

### PR DESCRIPTION
**Pull request by wrynvera**
Part of #23 
Silver and mithril are both added in as basic ores.
Their generation is copied off moreblocks, so there's no changes in that regard.

I added mithril tools to be something that is slightly worse than mese, but has 10x more durability.
I didn't incorporate copper rails because carts doesn't exist and silver tools because I feel that we already have enough tools and we don't need more redundant ones.
Silver should preferably be incorporated into more crafting recipes to make it useful.

### CODE REVIEW

**Comment by wrynvera** [comment]
Since this is a minor change and mostly pre-approved, i'll squash and merge this once I get 1 dev approval.

**Review by lumberjackgames — APPROVED** [review]

**Pull request merged**
